lib/util/pools.js

```diff
@@ -78,10 +78,8 @@ export function createPool(name, opts) {
       }
     },
 
-    freeAll: function() {
-      let allocatedLength = allocated.length;
-      let freeLength = free.length;
-      let reAlloc = allocated.slice(0, size - freeLength);
+    freeAll: function freeAllPool() {
```

Comment:
This is a good point. I should refactor this code. The reason why `Pool` was attached is that I was colliding with internal variables. Like `var protect = []` and then `function protect` would collide. This is easily fixed by updating the variable names to reflect what they are.

@jugglinmike had shown me the new naming semantics, but you're right I don't think they are actually implemented anywhere.